jeongjaein/FBMember/app/src/main/java/user/manage/fbmember/LoginActivity.kt

```diff
@@ -9,31 +9,32 @@ import kotlinx.android.synthetic.main.activity_login.*
 
 class LoginActivity : AppCompatActivity() {
 
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_login)
 
         val accountManager = AccountManager.getInstance()
 
         btnlogin.setOnClickListener{
-            val intent = Intent(this, MypageActivity::class.java)
+            val MypageIntent = Intent(this, MypageActivity::class.java)
```

Comment:
```suggestion
            val mypageIntent = Intent(this, MypageActivity::class.java)
```
변수명은 snake case로 짓는게 규칙입니다.

Reply:
혹스 감사합니다

Reply:
snake case가 아니라 camel case입니다. 잘못 전달했네요

Reply:
넴 감사함다